jetty-websocket/README.TXT

```diff
@@ -15,10 +15,10 @@ A TestServer and TestClient class are available, and can be run either directly
 imported), or from the command line with
 
 
-  java -cp jetty-aggregate/jetty-all/target/jetty-all-7.x.y.jar:jetty-distribution/target/distribution/lib/servlet-api-2.5.jar
+  java -cp jetty-aggregate/jetty-tutorial/target/jetty-tutorial-7.x.y.jar:jetty-distribution/target/distribution/lib/servlet-api-2.5.jar
```

Comment:
Should not be this `jetty-tutorial-10.x.y.jar`? And the servlet-api jar changed accordingly?

Reply:
yow, old doc.  I'll have to update it too.